reaktive/src/commonTest/kotlin/com/badoo/reaktive/testutils/TestObservableObserverExt.kt

```diff
@@ -10,6 +10,9 @@ fun <T> TestObservableObserver<T>.getOnNextEvent(index: Int): Event.OnNext<T> =
 fun <T> TestObservableObserver<T>.getOnNextValue(index: Int): T =
     getOnNextEvent(index).value
 
+fun <T> TestObservableObserver<T>.getOnNextValues(): List<T> =
```

Comment:
This is already being added in https://github.com/badoo/Reaktive/pull/54, please wait for it and then reuse.

Reply:
Sure. For now I have copy-pasted the implementation in #54. Once it is merged, I can rebase and update this PR.

Reply:
Merged :+1: